pkgs/desktops/gnome/extensions/update-extensions.py

```diff
@@ -48,28 +51,35 @@ def fetch_extension_data(uuid: str, version: str) -> Tuple[str, str]:
     uuid = uuid.replace("@", "")
     url: str = f"https://extensions.gnome.org/extension-data/{uuid}.v{version}.shell-extension.zip"
 
-    # Yes, we download that file three times:
+    # TODO remove when Vitals@CoreCoding.com version != 53
+    if url == 'https://extensions.gnome.org/extension-data/VitalsCoreCoding.com.v53.shell-extension.zip':
+        url = 'https://extensions.gnome.org/extension-data/VitalsCoreCoding.com.v53.shell-extension_v1BI2FB.zip'
+
+    # Yes, we download that file two times:
 
-    # The first time is for the maintainter, so they may have a personal backup to fix potential issues
+    # The first time is for the maintainer, so they may have a personal backup to fix potential issues
     # subprocess.run(
     #     ["wget", url], capture_output=True, text=True
     # )
 
-    # The second time, we extract the metadata.json because we need it too
-    with urllib.request.urlopen(url) as response:
-        data = zipfile.ZipFile(io.BytesIO(response.read()), 'r')
-        metadata = base64.b64encode(data.read('metadata.json')).decode()
-
-    # The third time is to get the file into the store and to get its hash
-    hash = subprocess.run(
+    # The second time, we add the file to store
+    process = subprocess.run(
         ["nix-prefetch-url", "--unpack", url], capture_output=True, text=True
-    ).stdout.strip()
+    )
+
+    # Get hash from nix-prefetch-url output
+    hash = process.stdout.strip()
+
+    # Get metadata.json from nix store
+    path = nix_store_pattern.search(process.stderr).group(0)
```

Comment:
The `stderr` is undocumented. Better use `--print-path`, which is [available since 1.11](https://github.com/NixOS/nix/commit/a3c4eb096456673b48f4fc35b76097d67e022141).